app/components/ui/calendar.tsx

```diff
@@ -0,0 +1,64 @@
+"use client"
```

Comment:
Since we're not using RSC in Remix, I think it's safe to remove the 'use client' directives

@jeeehaan

Reply:
updated mas @milhamm